From 12a576be512138592b97babc11aee2f43bda5d5a Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solusos.com>
Date: Thu, 5 Sep 2013 17:18:16 +0100
Subject: [PATCH] Lighten background colour of notify-osd

---
 src/defaults.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/defaults.c b/src/defaults.c
index 9088630..deb6cf7 100644
--- a/src/defaults.c
+++ b/src/defaults.c
@@ -117,7 +117,7 @@ enum
 #define DEFAULT_BUBBLE_HORZ_GAP       0.5f
 #define DEFAULT_BUBBLE_SHADOW_SIZE    0.7f
 #define DEFAULT_BUBBLE_SHADOW_COLOR  "#000000"
-#define DEFAULT_BUBBLE_BG_COLOR      "#131313"
+#define DEFAULT_BUBBLE_BG_COLOR      "#222222"
 #define DEFAULT_BUBBLE_BG_OPACITY    "#cc"
 #define DEFAULT_BUBBLE_HOVER_OPACITY "#66"
 #define DEFAULT_BUBBLE_CORNER_RADIUS 0.375f
-- 
1.7.10.4

